test/offline/offline_integration.js

```diff
@@ -157,4 +157,45 @@ describe('Offline', function() {
         .catch(fail)
         .then(done);
   });
+
+  it('stores, plays, and deletes protected content with a temporary license', function(done) {
```

Comment:
Please change these `it()` calls to `drm_it()`.  We only run DRM tests with the --drm flag, and `drm_it()` does the filtering for us.

We do it this way so that tests can still be run without solid network connectivity.  --drm enables DRM tests (some networking required for licensing) and --external enables external asset tests (solid network required to pull content from the internet).